distribution/changelog.txt

```diff
@@ -7,6 +7,7 @@
 - Feature: [OpenMusic#54] Added Progressive ride music style (feat. Approaching Nirvana).
 - Change: [#22230] The plugin/script engine is now initialised off the main thread.
 - Change: [#22251] Hide author info in the scenery window unless debug tools are active.
+- Fix: [#22208] Area edit cursors can disappear in some cases.
```

Comment:
We usually call these tool selection areas. There were particular cases involved, right? Just the map edge?

Reply:
> We usually call these tool selection areas.

Right. So change to "Tool selection areas can disappear in some cases."?

> There were particular cases involved, right? Just the map edge?

That's a different issue, and would be #22209 ;-)